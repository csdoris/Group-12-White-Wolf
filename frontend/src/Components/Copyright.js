import Typography from '@material-ui/core/Typography';
import Link from '@material-ui/core/Link';

export default function Copyright() {
    return (
        <Typography variant="body2" color="textSecondary" align="center">
            {'Copyright Â© '}
            <Link color="inherit" href="#">
                Weather you are ready
        </Link>{' '}
            {new Date().getFullYear()}
            {'.'}
        </Typography>
    );
}