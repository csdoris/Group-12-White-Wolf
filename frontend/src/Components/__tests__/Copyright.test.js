import renderer from 'react-test-renderer';
import Copyright from '../Copyright';

it('snapshot with no content matches', () => {
    const tree = renderer.create(<Copyright />).toJSON();
    expect(tree).toMatchSnapshot();
});