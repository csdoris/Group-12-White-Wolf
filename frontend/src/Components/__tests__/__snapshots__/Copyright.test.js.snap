// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot with no content matches 1`] = `
<p
  className="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary MuiTypography-alignCenter"
>
  Copyright Â© 
  <a
    className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorInherit"
    href="#"
    onBlur={[Function]}
    onFocus={[Function]}
  >
    Weather you are ready
  </a>
   
  2021
  .
</p>
`;
